packages/events/src/lib.rs

```diff
@@ -4,13 +4,36 @@
 //! different parts of an application. The API is designed to be simple to use while offering
 //! high performance in concurrent scenarios.
 //!
-//! # Example
+//! Both single-threaded and thread-safe variants are available:
+//! - [`Event<T>`], [`ByRefEventSender<T>`], [`ByRefEventReceiver<T>`] - Thread-safe variants
+//! - [`LocalEvent<T>`], [`ByRefLocalEventSender<T>`], [`ByRefLocalEventReceiver<T>`] - Single-threaded variants
+//!
+//! # Thread-safe Example
 //!
 //! ```rust
+//! use std::sync::Arc;
+//!
 //! use events::once::Event;
 //!
-//! // Create an event for passing a string message
-//! let event = Event::<String>::new();
+//! // Create a thread-safe event for passing a string message
+//! let event = Arc::new(Event::<String>::new());
```

Comment:
Unclear why this is an Arc. It seems it would work fine without the Arc wrapper, no? Verify that we are not adding unnecessary cruft to examples in the API documentation.